src/js/themes/base.d.ts

```diff
@@ -782,6 +782,9 @@ export interface ThemeType {
       margin?: MarginType;
       pad?: PadType;
     };
+    checkBox: {
+      pad: PadType;
```

Comment:
Should not be optional like other keys ?

Reply:
Same for `CheckBox` key.

Reply:
yes you are correct pushing up a fix now sorry for any inconvenience. Thanks for the catch!